builtin/rebase.c

```diff
@@ -52,6 +57,13 @@ struct rebase_options {
 	int preserve_merges;
```

Comment:
I just noticed this: `preserve_merges` is implied and superseded by `type`, right?